travelrec/recommendations_pipeline.py

```diff
@@ -19,6 +19,18 @@
     'activities': 75
 }
 
+class NoLocationsFoundError(Exception):
+    """Raised when no locations were found based on provided query"""
+    pass
+
+class NoNearbyCitiesWithTemperatureFoundError(Exception):
```

Comment:
I do not understand difference between `NoNearbyCitiesWithTemperatureFoundError` and `NoNearbyCitiesWithClimateFoundError`

Reply:
- `NoNearbyCitiesWithTemperatureFoundError `- when user provides no climate restrictions, but sparql query returns nothing (in that case there is a dummy filter `?yearMeanC >-200`)
- `NoNearbyCitiesWithClimateFoundError `- when user provides climate restrictions, but nothing is found (e.g. "cold places near Rome")

Reply:
Ok, I would just go with `NoNearbyCitiesFoundError`